contracts/nns/contract.go

```diff
@@ -573,20 +581,49 @@ func AddRecord(name string, typ recordtype.Type, data string) {
 	updateSoaSerial(ctx, tokenID)
 }
 
+// AddNeoRecord adds domain record of the Neo type for the specified address in
+// 20-byte hash format. The name MUST NOT be a TLD.
+func AddNeoRecord(name string, address interop.Hash160) {
```

Comment:
Check for address length please (`interop.Hash160Len`), https://github.com/neo-project/neo/issues/1891 is still here and unchecked values can be abused.